ftml/src/data/user_info.rs

```diff
@@ -32,6 +32,9 @@ pub struct UserInfo<'a> {
     /// The user's display name.
     pub user_name: Cow<'a, str>,
 
+    /// The user's karma, from 0-5.
+    pub user_karma: u8,
```

Comment:
Since there's only 5 values it should be an enum, but that's out of scope for this PR, I'll change this for you.

Reply:
what is this, ENUMS? I don't even have INTEGERS in MY JAVASCRIPT